src/page-form/components/Form.tsx

```diff
@@ -19,15 +21,37 @@ interface FormValues {
 
 const states = zcta.getStates();
 
-const Form: React.FC = () => {
+const Form: React.FC<{}> = () => {
```

Comment:
nit: Does adding the `<{}>` do anything? I previously removed them because it looked like we could.